src/server/terrain/structure_building_blocks.zig

```diff
@@ -0,0 +1,288 @@
+const std = @import("std");
+
+const main = @import("root");
+const ZonElement = main.ZonElement;
+const Blueprint = main.blueprint.Blueprint;
+const List = main.List;
+const AliasTable = main.utils.AliasTable;
+const Neighbor = main.chunk.Neighbor;
+const Block = main.blocks.Block;
+const parseBlock = main.blocks.parseBlock;
+const Degrees = main.rotation.Degrees;
+const hashInt = main.utils.hashInt;
+const NeverFailingAllocator = main.heap.NeverFailingAllocator;
+
+var arena = main.heap.NeverFailingArenaAllocator.init(main.globalAllocator);
+const arenaAllocator = arena.allocator();
+
+var structureCache: ?std.StringHashMapUnmanaged(StructureBuildingBlock) = null;
+var blueprintCache: ?std.StringHashMapUnmanaged([4]*BlueprintEntry) = null;
+var childrenToResolve: List(struct {parentId: []const u8, colorName: []const u8, colorIndex: usize, childIndex: usize, structureId: []const u8}) = undefined;
+
+const BlueprintEntry = struct {
+	blueprint: Blueprint,
+	info: Info,
```

Comment:
We have two fields here, `Info` has two fields, together the functions are less than 100 lines, `Info` is a pretty generic name.
I see no reason to have it as a seperate struct. I would just inline it here.

Reply:
Ok, will be your way.